<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolButton,QPushButton{background:transparent;}
</string>
  </property>
  <widget class="QWidget" name="bottomwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>920</width>
     <height>430</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget,QStackedWidget{background:white;}
QSplitter::handle{background:lightgray;}
QPushButton{color:rgb(0, 120,230);background:transparent;}
QPushButton:hover{color:rgb(100, 190, 250);}
</string>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>920</width>
      <height>430</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>5</number>
    </property>
    <widget class="QWidget" name="page_0">
     <widget class="QWidget" name="page0_mainwidget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>920</width>
        <height>430</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>920</width>
         <height>430</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="opaqueResize">
        <bool>true</bool>
       </property>
       <property name="handleWidth">
        <number>1</number>
       </property>
       <property name="childrenCollapsible">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="page0_leftwidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>600</width>
          <height>460</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QWidget" name="page0_rightwidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>320</width>
          <height>460</height>
         </size>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_1">
     <widget class="QWidget" name="page1_mainwidget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>920</width>
        <height>430</height>
       </rect>
      </property>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>150</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QWidget" name="page2_mainwidget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>920</width>
        <height>430</height>
       </rect>
      </property>
      <widget class="QCommandLinkButton" name="commandLinkButton">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>180</y>
         <width>187</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>CommandLinkButton</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QWidget" name="page3_mainwidget" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>920</width>
        <height>430</height>
       </rect>
      </property>
      <widget class="QCalendarWidget" name="calendarWidget">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>110</y>
         <width>296</width>
         <height>236</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QWidget" name="page4_mainwidget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>920</width>
        <height>430</height>
       </rect>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>150</y>
         <width>120</width>
         <height>80</height>
        </rect>
       </property>
       <property name="title">
        <string>GroupBox</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QWidget" name="page5_mainwidget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>920</width>
        <height>430</height>
       </rect>
      </property>
      <widget class="QProgressBar" name="progressBar">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>130</y>
         <width>201</width>
         <height>51</height>
        </rect>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="topwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>920</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{background:transparent;background-image:url(:/img/skin/skin.png);}
</string>
   </property>
   <widget class="QWidget" name="top_topwidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>920</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover{border-radius:5px; border:1px solid rgb(210, 225, 230);}
</string>
    </property>
    <widget class="QLabel" name="label_title">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>251</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
font: 15pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>360安全卫士</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_skin">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>0</y>
       <width>28</width>
       <height>28</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>28</width>
       <height>28</height>
      </size>
     </property>
     <property name="toolTip">
      <string>换肤</string>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="qrc.qrc">
       <normaloff>:/img/sysButton/skin_button.png</normaloff>:/img/sysButton/skin_button.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_min">
     <property name="geometry">
      <rect>
       <x>866</x>
       <y>0</y>
       <width>28</width>
       <height>28</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>28</width>
       <height>28</height>
      </size>
     </property>
     <property name="toolTip">
      <string>最小化</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="qrc.qrc">
       <normaloff>:/img/sysButton/min_button.png</normaloff>:/img/sysButton/min_button.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_close">
     <property name="geometry">
      <rect>
       <x>894</x>
       <y>0</y>
       <width>28</width>
       <height>28</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>28</width>
       <height>28</height>
      </size>
     </property>
     <property name="toolTip">
      <string>关闭</string>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="qrc.qrc">
       <normaloff>:/img/sysButton/close_button.png</normaloff>:/img/sysButton/close_button.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_menu">
     <property name="geometry">
      <rect>
       <x>838</x>
       <y>0</y>
       <width>28</width>
       <height>28</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>28</width>
       <height>28</height>
      </size>
     </property>
     <property name="toolTip">
      <string>设置</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="qrc.qrc">
       <normaloff>:/img/sysButton/main_menu.png</normaloff>:/img/sysButton/main_menu.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="top_btmwidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>920</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{color:white;}
QToolButton:hover{border-image:url(:/img/toolWidget/toolbar_hover.png)}
QToolButton:on{border-image:url(:/img/toolWidget/toolbar_hover.png)}</string>
    </property>
    <widget class="QToolButton" name="tbtn_serhome">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>90</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>服务大厅</string>
     </property>
     <property name="icon">
      <iconset resource="qrc.qrc">
       <normaloff>:/img/toolWidget/ico_service.png</normaloff>:/img/toolWidget/ico_service.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="tbtn_syscheck">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>0</y>
       <width>90</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>系统检测</string>
     </property>
     <property name="icon">
      <iconset resource="qrc.qrc">
       <normaloff>:/img/toolWidget/ico_control.png</normaloff>:/img/toolWidget/ico_control.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>0</y>
       <width>90</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>电脑清理</string>
     </property>
     <property name="icon">
      <iconset resource="qrc.qrc">
       <normaloff>:/img/toolWidget/qingLi.png</normaloff>:/img/toolWidget/qingLi.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>90</width>
       <height>90</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_3">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>0</y>
       <width>90</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>系统修复</string>
     </property>
     <property name="icon">
      <iconset resource="qrc.qrc">
       <normaloff>:/img/toolWidget/xiTong.png</normaloff>:/img/toolWidget/xiTong.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>90</width>
       <height>90</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_4">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>0</y>
       <width>90</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>优化加速</string>
     </property>
     <property name="icon">
      <iconset resource="qrc.qrc">
       <normaloff>:/img/toolWidget/jiaSu.png</normaloff>:/img/toolWidget/jiaSu.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>90</width>
       <height>90</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_5">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>0</y>
       <width>90</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>漏洞修复</string>
     </property>
     <property name="icon">
      <iconset resource="qrc.qrc">
       <normaloff>:/img/toolWidget/louDong.png</normaloff>:/img/toolWidget/louDong.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>90</width>
       <height>90</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="status_widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>550</y>
     <width>920</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{background:url(:/img/skin/skin.png)}
QLabel{color:white}</string>
   </property>
   <widget class="QLabel" name="label_verson">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>版本：1.0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_addr">
    <property name="geometry">
     <rect>
      <x>768</x>
      <y>0</y>
      <width>151</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="qrc.qrc"/>
 </resources>
 <connections/>
</ui>
